### ISSUE TITLE
update the website?

### ISSUE THREAD

**Issue by evertbouw** [redux-observable/redux-observable#789]
Is it worth fixing the website, should it be redone, or taken down?

Personally I'm fine with just readmes on github. I don't want to put too much effort into a website because in 2024 I'd recommend looking at a bunch of different solutions before reaching for redux middleware.

**Comment by MaximeBernard** [comment]
With Redux releasing a new major version, maybe it will regain a new audience.

Also, I'd be interested to know how many visitors the website has (I saw GTag). With 225,150 weekly downloads on NPM, some people maintaining legacy projects might still be happy with the current website. 

That being said, I agree we shouldn't put too much effort into it. I have no idea how the website is updated but maybe updating few MD files would be enough?

**Comment by evertbouw** [comment]
There were some scripts to turn the md files into a website, it's published by pushing to https://github.com/redux-observable/redux-observable/tree/gh-pages

**Comment by piyushchauhan2011** [comment]
I was wondering we can use docusaurus or vitepress and manage different redux-observable version docs efficiently. I would be happy to help

**Comment by evertbouw** [comment]
Yes some help to get a new docs site running would be very welcome. The text in the docs can use a rewrite but let's call that out of scope for now. Hopefully we can keep the same paths or at least redirect (for SEO?). We won't need a blog. The most fancy feature I can think of is an interactive example that can be an embedded codesandbox or MDX widget.

**Comment by MikeRyanDev** [comment]
We will be building and releasing an updated docs site in the next few months. Stay tuned!
